route-rs-runtime/Cargo.toml

```diff
@@ -8,4 +8,5 @@ license = "MIT"
 [dependencies]
 tokio = "*"
 futures = "*"
-crossbeam = "*"
\ No newline at end of file
+crossbeam = "*"
+smoltcp = "0.5.0"
```

Comment:
For now, let's let our dependencies float unless we have a good reason to lock them.